app/scripts/components/exploration/components/datasets/layer-chart-analysis-menu.tsx

```diff
@@ -4,38 +4,59 @@ import { Dropdown } from '@devseed-ui/dropdown';
 import AnalysisMetrics, { DataMetric } from './analysis-metrics';
 import { TipButton } from '$components/common/tip-button';
 
-
 interface ChartAnalysisProps {
   triggerIcon: JSX.Element;
   onChange: (changeType: string, item) => void;
+  variablesList?: string[];
+  setSelectedVariable?: (variable: string) => void;
   activeMetrics: DataMetric[];
 }
 
 const IconButton = styled(TipButton)`
   z-index: 1;
 `;
 
-export default function LayerChartAnalysisMenu ({activeMetrics, triggerIcon, onChange}: ChartAnalysisProps) {
+export default function LayerChartAnalysisMenu({
+  activeMetrics,
+  triggerIcon,
+  variablesList,
+  setSelectedVariable,
+  onChange
+}: ChartAnalysisProps) {
+  const showVariableDropdown = variablesList? variablesList.length > 1 : false;
   return (
-    <Dropdown
-      alignment='right'
-      direction='up'
-      triggerElement={(props) => (
-        <IconButton
-          tipContent='Analysis metrics'
-          variation='base-text'
-          size='small'
-          fitting='skinny'
-          {...props}
+    <>
+      {showVariableDropdown && setSelectedVariable && variablesList && (
+        <select
```

Comment:
heads-up that I tried using uswds component, but spacing doesn't work. We might need a bigger change of deprecating devseed ui so other elements of timeline can match.

<img width="425" height="181" alt="Screenshot 2025-07-15 at 2 35 46 PM" src="https://github.com/user-attachments/assets/29dc5fee-fe09-44e1-9465-820ce0029c8a" />